web/themes/custom/cybersecurity/cybersecurity.config.js

```diff
@@ -2,23 +2,65 @@ module.exports = {
   prefix: 'tw-',
   separator: '-',
   theme: {
-    extend: {},
+    extend: {
+      spacing: {
+        '5p': '.5%',
+        '10p': '1%',
+        '15p': '1.5%',
+        '20p': '2%',
+        '14': '3.5rem',
+        '60': '15rem',
+      },
+      colors: {
+        'primary': '#361c82',
+        'secondary': '#020354',
+        'heading': '#010247',
+        'mid-heading': '#010247',
+        'sidebar-left-parent': '#0a165c',
```

Comment:
Are these sidebar globals reused multiple times, if not just use the value in the sass file or consider making them more generic like 'separator': '#2451a2'

Reply:
This has been updated